src/main/webapp/jsp/instructorFeedbackEdit.jsp

```diff
@@ -36,6 +40,7 @@
   </c:if>
    <br>
   <input type="hidden" id="num-questions" value="${fn:length(data.qnForms)}">
+  <input type="hidden" name="${FEEDBACK_SESSION_ENABLE_EDIT}" value="${data.isToBeLoadedInEditMode}">
```

Comment:
Oops, I meant optional tag attribute. Take a look at the top of `feedbackSessionsForm.tag` for an example of a required attribute.